managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json

```diff
@@ -1,16 +1,16 @@
 {
-    "TARGET_PROJECT_ID": <PROJECT_ID>,
-    "CLOUD_REGION": <REGION_ID>,
-    "TARGET_ENTRY_GROUP_ID": <ENTRY_GROUP_ID>,
-    "CREATE_TARGET_ENTRY_GROUP": <true/false>,
-    "CLOUD_STORAGE_BUCKET_ID": <BUCKET_ID>,
-    "SERVICE_ACCOUNT": <SERVICE_ACCOUNT_ID>,
-    "ADDITIONAL_CONNECTOR_ARGS": [],
-    "CUSTOM_CONTAINER_IMAGE": <IMAGE>,
-    "IMPORT_JOB_SCOPE_ENTRY_TYPES": [],
-    "IMPORT_JOB_SCOPE_ASPECT_TYPES": [],
+    "TARGET_PROJECT_ID": PROJECT_ID,
+    "CLOUD_REGION": LOCATION_ID,
+    "TARGET_ENTRY_GROUP_ID": ENTRY_GROUP_ID,
+    "CREATE_TARGET_ENTRY_GROUP": CREATE_ENTRY_GROUP_BOOLEAN,
+    "CLOUD_STORAGE_BUCKET_ID": BUCKET_ID,
+    "SERVICE_ACCOUNT": SERVICE_ACCOUNT_ID,
+    "ADDITIONAL_CONNECTOR_ARGS": [ADDITIONAL_CONNECTOR_ARGUMENTS],
+    "CUSTOM_CONTAINER_IMAGE": CONTAINER_IMAGE,
+    "IMPORT_JOB_SCOPE_ENTRY_TYPES": [ENTRY_TYPES],
+    "IMPORT_JOB_SCOPE_ASPECT_TYPES": [ASPECT_TYPES],
     "IMPORT_JOB_LOG_LEVEL": "INFO",
-    "NETWORK_TAGS": [],
-    "NETWORK_URI": "",
-    "SUBNETWORK_URI": ""
- }
\ No newline at end of file
+    "NETWORK_TAGS": [TAGS],
+    "NETWORK_URI": "NETWORK",
```

Comment:
Leave network_uri and subnetwork_uri empty.

Reply:
Done. Also removed them from the .tfvars sample.

What about NETWORK_TAGS, should I leave that empty too?

Reply:
yes